week-2-wide-perspective/w02e03-pagination-problem/src/middlewares/pagination.js

```diff
@@ -0,0 +1,14 @@
+const paginationMiddleware =
+  ({ limitKey = 'limit', skipKey = 'skip' }) =>
+  (req, res, next) => {
+    console.log(req.query)
+    if (req.query[skipKey]) {
+      req.skip = Number(req.query[skipKey]);
+    }
+    if (req.query[limitKey]) {
+      req.limit = Number(req.query[limitKey]);
```

Comment:
Bardzo fajnie - dokładnie o to chodziło.
Jedyne co to moja sugestia.

Wrzuciłbym to w `namespace` w `req` np:
```javascript
req.pagination = { skip, limit }
```

takie wiesz, w razie czego - żeby nie zdublować nic w `req` a z `2` strony wszystko z middleware zebrać jeden obiekt jako `req` property.